tests/Nargo.toml

```diff
@@ -1,9 +1,10 @@
 [package]
-name = "merkle_trees"
+name = "zk_kit_tests"
 type = "lib"
-authors = ["fabianschu", "signorecello"]
-compiler_version = ">=0.39.0"
+authors = ["signorecello"]
 
 [dependencies]
-bignum = { git = "https://github.com/noir-lang/noir-bignum", tag = "zpedro/default_trait" }
+bignum = { git = "https://github.com/noir-lang/noir-bignum", tag = "main" }
```

Comment:
fyi, this will break pretty much immediately. Pulling whatever is latest on master seems like a really bad idea.

Reply:
Yeah should've picked a version. Let's see what happens